tests/elements-regression/tests/elements-regression.test.ts

```diff
@@ -68,15 +68,19 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 
 			await editorPage.loadTemplate( filePath, true );
 			await editorPage.waitForIframeToLoaded( widgetType );
-			await helper.doScreenshotComparison( { widgetType, hoverSelector } );
+
+			await page.setViewportSize( { width: 1920, height: 3080 } );
```

Comment:
I see that this viewport size is pretty unique,
But maybe we can add the screen sizes to the `viewportSize` const in `tests/playwright/assets/elements-utils.ts`, that way we will have one source of truth to all screen sizes.

Reply:
this case is pretty uniq. I need this big viewport in order to do a big screenshot with more then one widget. In all other place it's not needed.